realm/realm-jni/src/io_realm_internal_TableQuery.cpp

```diff
@@ -1523,42 +1521,44 @@ JNIEXPORT jdouble JNICALL Java_io_realm_internal_TableQuery_nativeAverageDouble(
 
 
 // date aggregates
-
-JNIEXPORT jobject JNICALL Java_io_realm_internal_TableQuery_nativeMaximumDate(
+// FIXME: This is a rough workaround while waiting for https://github.com/realm/realm-core/issues/1720 to be solved
```

Comment:
I'll update the comment to be a link to https://github.com/realm/realm-core/issues/1745